src/TreeContainer.js

```diff
@@ -330,18 +342,24 @@ export const AugmentedTreeContainer = connect(
         config: state.config,
     }),
     dispatch => ({dispatch}),
-    (stateProps, dispatchProps, ownProps) => ({
-        _dashprivate_dependencies: stateProps.dependencies,
-        _dashprivate_dispatch: dispatchProps.dispatch,
-        _dashprivate_layout: ownProps._dashprivate_layout,
-        _dashprivate_path: ownProps._dashprivate_path,
-        _dashprivate_loadingState: getLoadingState(
-            ownProps._dashprivate_layout,
-            stateProps.requestQueue
-        ),
-        _dashprivate_requestQueue: stateProps.requestQueue,
-        _dashprivate_config: stateProps.config,
-    })
+    (stateProps, dispatchProps, ownProps) => {
+        const registry = ownProps._dashprivate_registry || Registry;
```

Comment:
This will work, but perhaps it would be simpler to merge `registry` into `config` (in `AppContainer` per https://github.com/plotly/dash-renderer/pull/173#discussion_r284204150, including this fallback to `Registry`) which puts it in the `store`?